Add blocks field to SlackAlerterParameters for custom slack blocks (#2153)
zenml-io/zenml: docs/book/stacks-and-components/component-guide/alerters/slack.md

# Slack Alerter

The `SlackAlerter` enables you to send messages to a dedicated Slack channel
directly from within your ZenML pipelines.

The `slack` integration contains the following two standard steps:

* [slack\_alerter\_post\_step](https://sdkdocs.zenml.io/latest/integration\_code\_docs/integrations-slack/#zenml.integrations.slack.steps.slack\_alerter\_post\_step.slack\_alerter\_post\_step)

>   takes a string message or a custom [slack block](https://api.slack.com/block-kit/building), posts it to a Slack channel, and returns whether the

Reviewer: ```suggestion
  takes a string message or a custom [Slack block](https://api.slack.com/block-kit/building), posts it to a Slack channel, and returns whether the
```

  operation was successful.
* [slack\_alerter\_ask\_step](https://sdkdocs.zenml.io/latest/integration\_code\_docs/integrations-slack/#zenml.integrations.slack.steps.slack\_alerter\_ask\_step.slack\_alerter\_ask\_step)

>   also posts a message or a custom [slack block](https://api.slack.com/block-kit/building) to a Slack channel, but waits for user feedback, and

Reviewer: ```suggestion
  also posts a message or a custom [Slack block](https://api.slack.com/block-kit/building) to a Slack channel, but waits for user feedback, and
```

    my_pipeline()
```

> An example of adding a custom slack block to your pipeline could look like this:

Reviewer: ```suggestion
An example of adding a custom Slack block as part of any alerter logic for your pipeline could look like this:
```
